Overview.src.html

```diff
@@ -988,16 +976,23 @@ <h2 class="no-num no-toc">
                 display</span> for <em>availabilityUrl</em>. Otherwise, set
                 <em>newAvailability</em> to <code>false</code>.
                 </li>
-                <li>Set <em>A</em>'s <code>value</code> property to
-                <em>newAvailability</em>.
-                </li>
-                <li>If <em>P</em> is not settled, then
-                  <span data-anolis-spec="promguide" title=
-                  "resolve-reject">resolve</span> <em>P</em> with <em>A</em>.
-                </li>
                 <li>If <em>previousAvailability</em> is not equal to
-                <em>newAvailability</em>, <span>signal a presentation display
-                availability change</span> with <em>A</em>.
+                <em>newAvailability</em>, then <span data-anolis-spec=
+                "w3c-html">queue a task</span> to run the following steps:
+                  <ol>
+                    <li>Set <em>A</em>'s <code>value</code> property to
+                    <em>newAvailability</em>.
+                    </li>
+                    <li>
+                      <span data-anolis-spec="w3c-html" title=
+                      "concept-event-fire">Fire an event</span> named
```

Comment:
Perhaps this could be stated as either resolving the promise or firing the event, to make it very clear that firing the event wouldn't have any effect, so that we can avoid having the discussion of the ordering of promise and event.